it/controllers/expenses/OtherEquipmentAmountControllerISpec.scala

```diff
@@ -0,0 +1,530 @@
+  }
+
+  object CommonExpectedEN extends CommonExpectedResults {
+    val expectedCaption = s"Employment expenses for 6 April ${taxYearEOY - 1} to 5 April $taxYearEOY"
+    val continueButtonText = "Continue"
+    val hintText = "For example, £600 or £193.54"
+  }
+
+  object CommonExpectedCY extends CommonExpectedResults {
+    val expectedCaption = s"Employment expenses for 6 April ${taxYearEOY - 1} to 5 April $taxYearEOY"
+    val continueButtonText = "Continue"
+    val hintText = "For example, £600 or £193.54"
+  }
+
+  val userScenarios: Seq[UserScenario[CommonExpectedResults, SpecificExpectedResults]] = {
+    Seq(UserScenario(isWelsh = false, isAgent = false, CommonExpectedEN, Some(ExpectedIndividualEN)),
+      UserScenario(isWelsh = false, isAgent = true, CommonExpectedEN, Some(ExpectedAgentEN)),
+      UserScenario(isWelsh = true, isAgent = false, CommonExpectedCY, Some(ExpectedIndividualCY)),
+      UserScenario(isWelsh = true, isAgent = true, CommonExpectedCY, Some(ExpectedAgentCY)))
+  }
+
+  ".show" should {
+
+    userScenarios.foreach { user =>
+
+      import Selectors._
+      import user.commonExpectedResults._
+
+      s"language is ${welshTest(user.isWelsh)} and request is from an ${agentTest(user.isAgent)}" should {
+
+        "render 'How much do you want to claim for buying other equipment?' page with the correct content and" +
+          " no pre-filled amount no values pre-filled when no user data" which {
+
+          lazy val result: WSResponse = {
+            dropExpensesDB()
+            authoriseAgentOrIndividual(user.isAgent)
+            userDataStub(userData(fullEmploymentsModel(hmrcExpenses = Some(employmentExpenses(fullExpenses.copy(otherAndCapitalAllowances = None))))), nino, taxYearEOY)
+            insertExpensesCyaData(expensesUserData(isPrior = true, hasPriorExpenses = true,
+              fullExpensesCYAModel.copy(expenses = fullExpensesCYAModel.expenses.copy(otherAndCapitalAllowances = None))), userRequest)
+            authoriseAgentOrIndividual(user.isAgent)
+            urlGet(employmentExpensesAmountPageUrl(taxYearEOY), user.isWelsh, follow = false,
+              headers = Seq(HeaderNames.COOKIE -> playSessionCookies(taxYearEOY)))
+          }
+
+          implicit def document: () => Document = () => Jsoup.parse(result.body)
+
+          "has an OK status" in {
+            result.status shouldBe OK
+          }
+
+          titleCheck(user.specificExpectedResults.get.expectedTitle)
+          h1Check(user.specificExpectedResults.get.expectedHeading)
+          captionCheck(user.commonExpectedResults.expectedCaption)
+          elementNotOnPageCheck(wantToClaimSelector)
+          buttonCheck(user.commonExpectedResults.continueButtonText, continueButtonSelector)
+          textOnPageCheck(hintText, hintTextSelector)
+          textOnPageCheck(poundPrefixText, poundPrefixSelector)
```

Comment:
can add a check to make sure the input field is displayed on the page (inputFieldCheck)